Use kwallet interface definition from the kwallet package,
instead of using the out-of-date provided in qtkeychain package
see https://github.com/frankosterfeld/qtkeychain/issues/172

--- a/CMakeLists.txt	2020-09-08 15:13:16.000000000 +0200
+++ b/CMakeLists.txt	2020-11-13 13:50:56.648621533 +0100
@@ -169,7 +169,7 @@
 
     add_definitions(-DKEYCHAIN_DBUS=1)
     list(APPEND qtkeychain_SOURCES keychain_unix.cpp gnomekeyring.cpp libsecret.cpp plaintextstore.cpp)
-    qt_add_dbus_interface(qtkeychain_SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/org.kde.KWallet.xml kwallet_interface KWalletInterface)
+    qt_add_dbus_interface(qtkeychain_SOURCES /usr/share/dbus-1/interfaces/kf5_org.kde.KWallet.xml kwallet_interface KWalletInterface)
     list(APPEND qtkeychain_LIBRARIES ${QTDBUS_LIBRARIES} )
 endif()
 
